internal/router/router_test.go

```diff
@@ -32,6 +32,8 @@ type RouterSuite struct {
 
 var _ = gc.Suite(&RouterSuite{})
 
+var newResolvedURL = MustNewResolvedURL
```

Comment:
it makes the tests match the tests in other packages, where we use newResolvedURL rather than router.MustNewResolvedURL because it's more concise and it's something we do a lot. will change if you care :)